execution_engine/src/runtime/mod.rs

```diff
@@ -434,11 +434,48 @@ where
             .map_err(|e| ExecError::Interpreter(e.into()).into())
     }
 
+    /// Writes requested field from runtime context's block info to dest_ptr in the Wasm memory.
+    fn get_block_info(&self, field_idx: u8, dest_ptr: u32) -> Result<(), Trap> {
+        if field_idx == 0 {
```

Comment:
Do these literals correspond to the field index constans provided in runtime.rs?

Reply:
they correspond but one is on the host the other gets turned into wasm and is run from within a vm instance. caller / callee. could put consts into a shared crate (types) and use in both places but it doesn't amount to any _functional_ difference.